src/sidebar/components/annotation-body.js

```diff
@@ -28,7 +28,7 @@ import TagList from './tag-list';
  * @prop {string} text -
  *     The markdown annotation body, which is either rendered as HTML (if `isEditing`
  *     is false) or displayed in a text area otherwise.
- * @prop {MergedConfig} [settings]
+ * @prop {MergedConfig} settings
```

Comment:
Are all injected services in fact required? Or rather, are there any cases of a service prop existing where its not being used?

Reply:
> Are all injected services in fact required?

Yes. The service injection helper (`withServices`)  has no notion of optional services. It is however optional whether the consumer of the component overrides the service that `withServices` would otherwise inject. Currently only tests actually override the service.